guides/plugins/plugins/administration/using-utils.md

```diff
@@ -0,0 +1,59 @@
+# Using Utils
+
+Utils in the Shopware 6 Administration are registered to the Shopware object and are therefore accessible everywhere in the Administration.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. Of course you'll have to understand JavaScript. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Accessing the Utils
+
+As noted in the introduction utils in the Shopware 6 Administration are bound to the Shopware global object. Learn more about that [here](./the-shopware-object.md).
+
+In this very short example we'll show you how to use a function of utils, in this case the `deepCopy` function.
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/swag-basic-example/index.js" %}
+```javascript
+const { Component, Utils } = Shopware;
+
+Component.register('swag-basic-example', {
+
+    data: function () {
+        return {
+            testObject: {
+                testValue0: 0,
+                testValue1: 1,
+                testValue2: 2
+            },
+            clonedObeject: {
+
+            }
+        }
+    },
+
+    created: function () {
+        this.clone()
+    },
+
+    methods: {
+        clone() {
+            this.clonedObeject = Utils.object.cloneDeep(this.testObject);
+        }
+    }
+});
+```
+{% endcode %}
+
+## An overview of all the functions
```

Comment:
How many functions are there? I'd suggest using a table *in this article* to introduce all functions in a short manner, not just refering to a file. So the reader doesn't need to switch between articles and we don't get a direct dependency on the file in the core.

Reply:
about 50 or so and i think that is way to many to just, let alone explain

Reply:
I agree but please consider my remark I wrote to you in Slack, after all.

Reply:
I see why it is difficult to keep them listed here but I think you should at least list a few topics like hashing, type detection, currencies, file access, ... this will make it more likely to find this page wihtin a search.

Reply:
How did we ended up here, now? @PaddyS were you able to help yet?

Reply:
i wrote the reference in the past 2 days